raftify/src/config.rs

```diff
@@ -20,6 +20,8 @@ pub struct Config {
     pub snapshot_interval: Option<f32>,
     pub restore_wal_from: Option<u64>,
     pub restore_wal_snapshot_from: Option<u64>,
+
+    pub omit_heartbeat_log: bool,
 }
```

Comment:
Since we will be modifying the raft-rs's config (RaftConfig) anyway, there is no need to add `omit_heartbeat_log` to the raftify Config type.

We can simply use `RaftConfig.omit_heartbeat_log`